core/src/main/java/com/linecorp/armeria/server/annotation/ResponseConverterFunctionProvider.java

```diff
@@ -31,15 +30,10 @@ public interface ResponseConverterFunctionProvider {
 
```

Comment:
Could you add this to the Javadoc of this interface?
` * @see DelegatingResponseConverterFunctionProvider`